Framework/Algorithms/src/SumSpectra.cpp

```diff
@@ -146,6 +149,13 @@ void SumSpectra::init() {
                   "estimate the number of dropped values see the "
                   "description. ");
 
+  declareProperty("MultiplyBySpectra", true,
```

Comment:
Is this going to break the existing api if anyone has called `SumSpectra` with positional arguments in Python?

Reply:
I thought we had been telling people to use named arguments for a few years.

Reply:
We'd been recommending it but the api does still allow `*args`. As `SumSpectra` is a widely used algorithm I think we could conceivable have uses of this in scripts we don't know about.